assets/scss/components/_service-info.scss

```diff
@@ -0,0 +1,194 @@
+/*
+Service Info
+
+Service Info provides an area to provide details about the service.
+
+Markup:
+<div class="service-info">
+  <div class="logo">
+    <span class="organisation-logo organisation-logo-medium">
+    HM Revenue &amp; Customs
+    </span>
+  </div>
+</div>
+
+Styleguide Service Info
+*/
+
+/*
+Logged In
+
+Experimental: This example shows a service info element with login details. This component needs a refactor to combine
+the multiple classes into a component. It is not recommended to use it in its current state.
+
+Markup:
+<div class="service-info ">
+  <div class="logo">
+    <span class="organisation-logo organisation-logo-medium">
+    HM Revenue &amp; Customs
+    </span>
+  </div>
+  <div class="last-login">
+    <p class="text--right h3-heading">John Smith</p>
+    <p class="last-login__more-details text--right font-xsmall js-visible"
+       data-toggle-details="last-login-details">
+    Last sign in: 31 May 2016 - <a href="#">not right?</a>
+    </p>
+  </div>
+  <section class="last-login-details alert alert--slim alert--info js-hidden">
+    <a class="minimise js-visible" href="#">Minimise</a>
+    <p class="alert__message">
+    Last sign in: Tuesday, 31 May 2016 at 2:44pm
+    </p>
+    <p class="alert__message">
+      <a href="https://www.gov.uk/hmrc-online-services-helpdesk">
+      Contact HMRC</a> if it wasn't you.
+    </p>
+  </section>
+</div>
+
+Styleguide Service Info.Logged In
+*/
+
+/*
+Help
+
+Service Info with `.service-info__help` provides a section to provide help information.
+
+Markup:
+<div class="service-info">
+  <h2>Service Information</h2>
+  <div class="service-info__help">
+    <p>You last signed in: Tuesday, 31 May 2016 at 2:44pm
+    <br><a href="#">Sign out</a>
+    </p>
+  </div>
+</div>
+
+Styleguide Service Info.With Help
+*/
+
+
+/*
+Agent Name
+
+Service Info with `.service-info__agent-name` provides highlighting for `.acting-agent`
+
+Markup:
+<div class="service-info">
+  <div class="logo">
+    <span class="organisation-logo organisation-logo-medium">
+    HM Revenue &amp; Customs
+    </span>
+  </div>
+  <div class="service-info__agent-name">
+    <p class="agent-name">
+      <span>Agent Name</span>
+      <span class="acting-agent"><a href="#">link</a></span>
```

Comment:
Maybe update this link to say `Account link` so it's more obvious what it's intention is